### DISCUSSION TITLE
Brand illustrations in Dark mode

### DISCUSSION THREAD

**Discussion by sharfatehi** [twilio-labs/paste#1875] [General] [answered]
Hello! I am using the brand illustration for an empty state in Flex and the illustration mostly disappears in dark mode. Are there any guidlines around using the illustrations in dark mode?

![Screen Shot 2021-09-16 at 3 21 31 PM](https://user-images.githubusercontent.com/78043210/134021307-949722c2-1734-49b1-84ad-ffaf99547129.png)

**Comment by TheSisb** [comment, marked as answer]
Hi @sharfatehi,

Dark mode is still in an early preview stage and so the illustrations don't look as nice as we'd like them to at this time.  This is outlined in [the Dark Mode RFC thread](https://github.com/twilio-labs/paste/discussions/1527):

> What about illustrations?
> Our illustrations are designed for Light Mode. They don’t adapt to dark backgrounds and as such they don’t look as good as we would like them to. To make illustrations work best in both Light and Dark Modes, we will likely have to rethink how illustrations are built. We will share more details on that in the future and have guidance when we graduate from an early preview.
